changelogs/unreleased/4315-env-scenarios.yml

```diff
@@ -0,0 +1,6 @@
+change-type: patch
+description: 'Enviroment e2e test scenarios'
+issue-nr: 4315
+destination-branches:
+- master
+- iso6
```

Comment:
you'll need to add iso5 too, sorry don't remember if I mentioned it, but the bot will not merge it otherwise :D